dts/arm/microchip/mec172xnsz.dtsi

```diff
@@ -825,7 +825,7 @@
 			mbox0: mbox@400f0000 {
 				compatible = "microchip,xec-espi-host-dev";
 				reg = <0x400f0000 0x200>;
-				interrupts = <60 0>;
+				interrupts = <60 3>;
```

Comment:
The commit message doesn't explain why you're changing interrupt priorities.

Reply:
will add this in the commit message
"The espi peripherals don't require the maximum priority hence they are being made uniform and a lower priority."

Reply:
updated